### PR TITLE
Add russian translates and update readme

### PR DESCRIPTION
**Pull request by akellbl4**

### CODE REVIEW

File: frontend/app/locales/ru.json [hunk 1]
```diff
@@ -78,33 +78,33 @@
   "commentsSort.worst": "Худшие",
   "emailLoginForm.back": "Назад",
   "emailLoginForm.confirm": "Подтвердить",
-  "emailLoginForm.email-address": "Email Address",
-  "emailLoginForm.empty-token": "Token field must not be empty",
-  "emailLoginForm.expired-token": "Token is expired",
-  "emailLoginForm.invalid-email": "Address should be valid email address",
-  "emailLoginForm.loading": "Loading...",
+  "emailLoginForm.email-address": "Email адресс",
```
Comment (paskal) [inline frontend/app/locales/ru.json]:
```suggestion
  "emailLoginForm.email-address": "Email адрес",
```

File: README.md [hunk 2]
```diff
@@ -472,7 +472,7 @@ window.REMARK42.changeTheme('light');
 
 ##### Locales
 
-Right now Remark has support three locales en, ru (partial translated), de(not translated).
+Right now Remark has support three locales en, ru (partial translated), de and fi.
```
Comment (paskal) [inline README.md]:
```suggestion
Right now Remark has support three locales en, ru, de and fi.
```

File: frontend/app/locales/ru.json [hunk 3]
```diff
@@ -52,7 +52,7 @@
   "comment.verified-user": "Подлинная учетная запись",
   "comment.verify-user": "Подвердить подлинность учетной записи для {userName}?",
   "comment.vote-error": "Ошибка голосования: {voteErrorMessage}",
-  "commentForm.anonymous-uploading-disabled": "Image uploading is disabled for unauthorized users. You should login before uploading.",
+  "commentForm.anonymous-uploading-disabled": "Загрузка изображений не доступна для не авторизированных пользователей. Авторизуйтесь для загрузки изображений.",
```
Comment (paskal) [inline frontend/app/locales/ru.json]:
```suggestion
  "commentForm.anonymous-uploading-disabled": "Загрузка изображений не доступна для неавторизированных пользователей. Авторизуйтесь для загрузки изображений.",
```

File: frontend/app/locales/ru.json [hunk 4]
```diff
@@ -78,33 +78,33 @@
   "commentsSort.worst": "Худшие",
   "emailLoginForm.back": "Назад",
   "emailLoginForm.confirm": "Подтвердить",
-  "emailLoginForm.email-address": "Email Address",
-  "emailLoginForm.empty-token": "Token field must not be empty",
-  "emailLoginForm.expired-token": "Token is expired",
-  "emailLoginForm.invalid-email": "Address should be valid email address",
-  "emailLoginForm.loading": "Loading...",
+  "emailLoginForm.email-address": "Email адресс",
+  "emailLoginForm.empty-token": "Поле токен должно не должно быть пустым",
```
Comment (paskal) [inline frontend/app/locales/ru.json]:
```suggestion
  "emailLoginForm.empty-token": "Поле ввода токена должно не должно быть пустым",
```

File: frontend/app/locales/ru.json [hunk 5]
```diff
@@ -78,33 +78,33 @@
...
+  "emailLoginForm.expired-token": "Время действия токена истекло",
+  "emailLoginForm.invalid-email": "Введен не корректный email адрес",
```
Comment (paskal) [inline frontend/app/locales/ru.json]:
```suggestion
  "emailLoginForm.invalid-email": "Введен некорректный email адрес",
```

File: frontend/app/locales/ru.json [hunk 6]
```diff
@@ -131,23 +131,23 @@
...
+  "subscribeByEmail.subscribe-to-replies": "Подписаться на ответы",
+  "subscribeByEmail.subscribed": "Вы были подписаны на обновления по email",
+  "subscribeByEmail.token": "Токен",
   "subscribeByEmail.unsubscribe": "Отказаться от подписки",
   "subscribeByRSS.button-title": "Подписаться по RSS",
   "subscribeByRSS.replies": "Ответы",
-  "subscribeByRSS.site": "Site",
-  "subscribeByRSS.thread": "Thread",
+  "subscribeByRSS.site": "Сайт",
+  "subscribeByRSS.thread": "Тред",
```
Comment (paskal) [inline frontend/app/locales/ru.json]:
```suggestion
  "subscribeByRSS.thread": "Ветка",
```

File: README.md (hunk 2 above)
Comment (akellbl4) [reply README.md]: Russian translation isn't completed yet.

File: frontend/app/locales/ru.json [hunk 7]
```diff
@@ -78,33 +78,33 @@
   "commentsSort.worst": "Худшие",
   "emailLoginForm.back": "Назад",
   "emailLoginForm.confirm": "Подтвердить",
-  "emailLoginForm.email-address": "Email Address",
-  "emailLoginForm.empty-token": "Token field must not be empty",
-  "emailLoginForm.expired-token": "Token is expired",
-  "emailLoginForm.invalid-email": "Address should be valid email address",
-  "emailLoginForm.loading": "Loading...",
+  "emailLoginForm.email-address": "Email адрес",
+  "emailLoginForm.empty-token": "Поле ввода токена должно не должно быть пустым",
```
Comment (sikmir) [inline frontend/app/locales/ru.json]: > должно не должно

**Pull request merged**